src/components/World/index.tsx

```diff
@@ -441,7 +447,7 @@ class World extends React.PureComponent<Props & ReduxWorldProps, State> {
     const itemsName = StyledText.compose({
```

Comment:
Item names themselves aren't being translated, possible due to hardcoded `EN_US` below:

```
itemList.push(EditableList.Item.standard({
        component: Item,
        props: { name: node.name[LocalizedString.EN_US], theme },
```